guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p>@jyaistMap for MLModel and FullyConnectedModel, can you change the descriptions to:</p><p> </p><ul><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#fullyconnectednetwork" target="_blank">FullyConnectedNetwork</a></code>&nbsp;- fully connected neural network for tabular/feature data</li><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#mlmodel" target="_blank">MLModel</a></code>&nbsp;- integrates Scikit-learn's machine learning models</li><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#prepare-tabulardata" target="_blank">prepare_tabulardata()</a></code>&nbsp;function for data preperation</li></ul><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Changed those descriptions:</p><p> </p><p>* Adds models to module:</p><p>&nbsp;* <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#fullyconnectednetwork" target="_blank">FullyConnectedNetwork</a> - fully connected neural network for tabular/feature data</p><p>&nbsp;* [<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#mlmodel" target="_blank">MLModel</a></code>]() - integrates Scikit-learn's machine learning models</p><p>&nbsp;* [<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#superesolution" target="_blank">SuperResolution</a></code>]()</p><p>&nbsp;* [<code>YOLOv3</code>]()</p><p>&nbsp;* [<code>FasterRCNN</code>](https://developers.arcgis.com/python/api-reference/arcgis.learn.html#fasterrcnn)</p><p>* Adds [<code>prepare_tabulardata()</code>](https://developers.arcgis.com/python/api-reference/arcgis.learn.html#prepare-tabulardata) function for data preperation</p>

Reply:
<p>Made those changes, <a href="https://github.com/rohitgeo" target="_blank">rohitgeo</a>. Thanks for looking through it.</p>